### PR TITLE
HYTV-174: Drupal 10 upgrade.

### PR DESCRIPTION
**Pull request by deiviic**
https://opetustila-test.it.helsinki.fi/

### CODE REVIEW

File: drupal/patches/d10_version_support.patch [hunk 1]
```diff
@@ -0,0 +1,14 @@
+diff --git a/translatable_menu_link_uri.info.yml b/translatable_menu_link_uri.info.yml
```
Comment (misterjoonas) [inline drupal/patches/d10_version_support.patch:1]: This module has D10 support now in 2.1.0 version https://www.drupal.org/project/translatable_menu_link_uri

File: drupal/patches/null_offset_fix.patch [hunk 2]
```diff
@@ -0,0 +1,14 @@
+diff --git a/core/modules/views/src/Plugin/views/query/QueryPluginBase.php b/core/modules/views/src/Plugin/views/query/QueryPluginBase.php
```
Comment (misterjoonas) [inline drupal/patches/null_offset_fix.patch:1]: Why is this patch applied? Is there an issue to reference? If not then one could be created against Drupal core and patch provided there.

Comment (misterjoonas) [reply drupal/patches/null_offset_fix.patch:1]: Check this issue https://www.drupal.org/project/drupal/issues/3361177

**Comment by misterjoonas** [comment]
There are a few updates available for contrib modules at https://opetustila-test.it.helsinki.fi/en/admin/reports/updates and Leaflet 10.0.16 seems to be unsupported.

**Review by misterjoonas — CHANGES_REQUESTED** [review]
A few more module updates and this should be good to go 👍

**Comment by deiviic** [comment]
@misterjoonas Anything else requires changes?

File: drupal/patches/d10_version_support.patch (hunk 1 above)
Comment (misterjoonas) [reply drupal/patches/d10_version_support.patch:1]: I guess the this file can be removed?

**Review by misterjoonas — APPROVED** [review]

**Pull request merged**